OlinkAnalyze/R/Olink_bridgeselector.R

```diff
@@ -59,6 +59,36 @@ olink_bridgeselector<-function(df, sampleMissingFreq, n){
                                       IQR < iqr_high,
                                     0, 1)) %>%
     dplyr::select(SampleID, Index, Panel, Outlier)
+  
+  
+  # Alternative LODs for when LOD is not present
+  if(!("LOD" %in% names(df))){
```

Comment:
So based on the info I got from the meeting, the files will have both Plate LOD and Max LOD values. If this is the case then the it will never step into the else if argument of Plate LOD %in% names, due to Max LOD always exisiting. 
And if it did it would overwrite the LOD value from row 68. 

Am I missing something here?